.github/workflows/ci.yml

```diff
@@ -138,13 +138,32 @@ jobs:
     timeout-minutes: 5
     name: "Checks: Helm tests"
     runs-on: ubuntu-latest
-    needs: [build-info]
+    needs: [build-info, ci-images]
+    env:
+      SKIP: "pylint"
+      MOUNT_LOCAL_SOURCES: "true"
+      PYTHON_MAJOR_MINOR_VERSION: ${{needs.build-info.outputs.defaultPythonVersion}}
     if: >
       needs.build-info.outputs.needs-helm-tests == 'true' &&
       (github.repository == 'apache/airflow' || github.event_name != 'schedule')
     steps:
       - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
         uses: actions/checkout@v2
+      - name: "Setup python"
```

Comment:
This seems redundant to me as we already have Python downloaded in the container.

@potiuk Can you help with CI setup?